bin/tictactoe

```diff
@@ -1,3 +1,56 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+#require "pry"
```

Comment:
2 Things

1: I would remove the pry from your final PR (makes it looks a lot cleaner)

2. I think abstracting line 5 - 56 into a separate class in the `lib` folder would be a great idea, and then on line 5 you could just call 
```ruby
CLI.new.start
```